library/Icingadb/Widget/Detail/CheckStatistics.php

```diff
@@ -124,7 +134,7 @@ protected function assembleBody(BaseHtmlElement $body)
             }
         }
 
-        $progressBar->getAttributes()->add('style', sprintf('width: %s%%', $leftNow));
+        $this->style->addFor($progressBar, ['width' => $leftNow . '%']);
```

Comment:
Use `sprintf('%F%%', ..` here, PHP7.x may use a comma as decimal point otherwise if the locale is e.g. de_DE